src/pages/en/guides/server-side-rendering.md

```diff
@@ -108,9 +108,16 @@ if(!product) {
 </html>
 ```
 
-#### API Routes
+## `API Routes`
+
+[API Routes](https://medium.com/@rajat_m/what-are-restful-routes-and-how-to-use-them-929129ae7bf6) is a powerful feature of SSR, represented by a `.js` or `.ts` file within the `src/pages` folder. They take a [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) and return a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response), while begin able to securely execute code on the server side.
```

Comment:
```suggestion
An [API Route](https://medium.com/@rajat_m/what-are-restful-routes-and-how-to-use-them-929129ae7bf6) is a `.js` or `.ts` file within the `src/pages` folder that takes a [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) and returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). A powerful feature of SSR, API routes are able to securely execute code on the server side.
```
Would this be correct to say? I do like the original's a clear definition of "a file here that takes this and returns this."

(And, if not, there are a couple of typos here: "API Routes is" and "begin" should be "being" ?)